### ISSUE TITLE
Install instructions no longer work?

### ISSUE THREAD

**Issue by nertia** [traxium/tabtree#174]
Hi, 

Following the instructions on the github 'Code' page to install the extension in the profile directory or by creating a zip fails on firefox 49.01 Debian stretch 64 bit. Nvidia graphics. 
The Firefox addons page version functions perfectly.

 It seems that newer versions of Firefox will no longer honor the "xpinstall.signatures.required false" setting for extensions even with "xpinstall.whitelist.required false" setting . From what I have read this will now only work with Developer versions of Firefox

I was just tinkering, hoping to tweak the theme colour settings directly from the css file although I now realise that this was probably not possible anyway due to manifests etc. I really don't know what I am doing.

This is from an attempted install following the first instructions without any alterations:
`1477154885736   addons.xpi-utils        WARN    addMetadata: Add-on TabsTree@traxium is invalid: Error: Directory /home/elijah/.mozilla/firefox/bxnq464t.default/extensions/TabsTree@traxium does not contain a valid install manifest (resource://gre/modules/addons/XPIProvider.jsm:1332:11) JS Stack trace: loadManifestFromDir<@XPIProvider.jsm:1332:11 < TaskImpl_run@Task.jsm:319:40 < TaskImpl@Task.jsm:280:3 < createAsyncFunction/asyncFunction@Task.jsm:254:14 < loadManifestFromFile@XPIProvider.jsm:1482:12 < syncLoadManifestFromFile@XPIProvider.jsm:1493:3 < addMetadata@XPIProviderUtils.js:1650:21 < processFileChanges@XPIProviderUtils.js:2014:23 < this.XPIProvider.checkForChanges@XPIProvider.jsm:3716:34 < this.XPIProvider.startup@XPIProvider.jsm:2706:25 < callProvider@AddonManager.jsm:236:12 < _startProvider@AddonManager.jsm:787:5 < AddonManagerInternal.startup@AddonManager.jsm:971:9 < this.AddonManagerPrivate.startup@AddonManager.jsm:2965:5 < amManager.prototype.observe@addonManager.js:71:9
`

I also tried the second method, creating my own zip from the github files, again without any alteration. This resulted in an extension corrupt message, using 'install from file' on the addons page.

Your extension is excellent and  I am grateful for any time you find to spend on it.

**Comment by traxium** [comment]
Yes. [With the release of Firefox 48, extension signing can no longer be disabled in the release and beta channel builds by using a preference.](https://blog.mozilla.org/addons/2016/07/29/extension-signing-availability-of-unbranded-builds/) So since Firefox 48 you have to use Firefox Developer Edition or [unbranded Firefox builds](https://wiki.mozilla.org/Add-ons/Extension_Signing#Unbranded_Builds) (but you still have to change xpinstall.signatures.required to false).
P.s. in order to just tweak Tab Tree css you could use [Stylish](https://addons.mozilla.org/en-US/firefox/addon/stylish/) without any hustle with Firefox installations.

**Comment by traxium** [comment]
E.g. it should make the rows much bigger:

``` CSS
@namespace url("http://www.mozilla.org/keymaster/gatekeeper/there.is.only.xul");

#tt-treechildren::-moz-tree-row,
#tt-treechildren-feedback::-moz-tree-row {
    min-height: 18px;
    height: 6em;
}
```
